apache/superset
# chore: Update font-sizes in ReportModal

### SUMMARY
This PR removes hacky usages of font-sizes by replacing them with the exact font-sizes available in the theme config.

### BEFORE

<img width="893" alt="Screen Shot 2022-04-07 at 17 56 16" src="https://user-images.githubusercontent.com/60598000/162229197-1e19df41-5227-4217-9e5b-ce0605a6ea99.png">

<img width="977" alt="Screen Shot 2022-04-07 at 18 00 50" src="https://user-images.githubusercontent.com/60598000/162230043-21241d01-989b-49fe-bb9c-df170027ae78.png">

### AFTER

<img width="1155" alt="reportmodal-after" src="https://user-images.githubusercontent.com/60598000/162229236-e1a18069-11b1-4e13-866b-9d605d8518d0.png">

<img width="856" alt="Screen Shot 2022-04-07 at 18 02 05" src="https://user-images.githubusercontent.com/60598000/162230254-e450f3da-7b63-48ad-b5e6-112a36cb0ef9.png">

### TESTING INSTRUCTIONS

1. Open the Report Modal from a Dashboard
2. Make sure the font-sizes look balanced

### ADDITIONAL INFORMATION
